### ISSUE TITLE
NTR: Anti-streptococcus DNAse B test positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8082]
New term request Anti-streptococcus DNAse B test positive (hausma_130829191812): Streptococcal  UMLS: 1168362

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000732
- label: Anti-streptococcus deoxyribonuclease-B antibody positivity
- parents: [Positive bloodstream antibacterial antibody positivity (HP:0430061)](https://hpo.jax.org/app/browse/term/HP:0430061)
- pmid: PMID:6752453
- def: The presence of antibodies in the blood circulation that react against streptococcal deoxyribonuclease-B (ADN-B). Almost all group A streptococci have this antigen and a strong elevation of the antibody is observed in patients with acute rheumatic fever, acute glomerulonephritis and other streptococcal infections.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8082
- synonyms:
